fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1125,18 +1125,8 @@ private void computeFragmentHosts() throws Exception {
                 if (dopAdaptionEnabled) {
```

Comment:
Do we still need this if branch?

Reply:
Yes, it should be reserved, here are two reasons:

1. if dop adaption enable, `PlanFragment::pipelineDop` is initialized to 1 in the constructor, and it has to be reassigned here
2. update local variable `maxParallelism`